### DISCUSSION TITLE
openagent for onelake

### DISCUSSION THREAD

**Discussion by georgehu0815** [xlang-ai/OpenAgents#114] [Show and tell]
add support for openagent to support delta lakehouse

1. add delta open table format for big data
2. add onelake python code tool
3. add onelake spark with pyspark auto script for delta table analytsis
4. add spark  agent for big data analysis
5. add UI support for result from spark dataframe
![image](https://github.com/xlang-ai/OpenAgents/assets/116222457/a56a7989-8694-4790-afeb-67031f37e7c8)

**Comment by BlankCheng** [comment]
Cool! Would you like to make a PR and test the new functionalities? I think it has been a huge effort.

**Reply by georgehu0815** [reply]
would like to add this feature, but i am facing issue with the old version langchain, can you please create branch with new version of langchian and I can go from there?
